app/Http/Controllers/ContactController.php

```diff
@@ -2,12 +2,34 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
+use App\Http\Requests\ContactRequest;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Mail\Message;
 
 class ContactController extends Controller
 {
-    public function index()
-    {
+    public function view(){
         return view('pages.contact');
     }
+    public function send(ContactRequest $request): RedirectResponse
+    {
+        $data = $request->validated();
+        \Log::debug('test', $data);
+        \Mail::send(
+            'molecules.emailContact',
+            [
+                'email' => $data['email'],
+                'name' => $data['name'],
```

Comment:
I requested a select value in the process.

Reply:
Added a select value "Subject" into contact form logic.

Reply:
accepted